chore(project): update readme (#434)
Informasjonsforvaltning/fdk-frontend: README.md

# FDK Frontend

> FDK Frontend is a monorepo for the frontend of the Felles datakatalog (FDK) also known as data.norge.no and other 

Author: Hva tenker du om noe sånt?

```suggestion
Monorepo powering the web frontend for data.norge.no. Built with NX, it includes the main Next.js web application, resuable React UI components, a Storybook component browser, and other related libraries.
```

> related projects. The monorepo is built using [Nx](https://nx.dev/) and contains several applications and libraries. 

Author: ```suggestion

```

> The applications are built using [Next.js](https://nextjs.org/) and the libraries are built using [React](https://reactjs.org/).

Author: ```suggestion
```

For a broader understanding of the system’s context, refer to the [architecture documentation](https://github.com/Informasjonsforvaltning/architecture-documentation) wiki. For more specific
context on this application, see the [Portal](https://github.com/Informasjonsforvaltning/architecture-documentation/wiki/Architecture-documentation#portal) subsystem section.

## Getting started

> This section describes how to get started with the FDK Frontend monorepo. It includes instructions for setting up the

Author: Disse setningene er overflødige.

```suggestion
```

> development environment, running the applications, and building the applications for production. 

Author: ```suggestion
```

### Prerequisites
- [Node.js](https://nodejs.org/en/download/) >=20.11.1
- [yarn](https://yarnpkg.com/getting-started/install) >=1.22.19

> - [git](https://git-scm.com/downloads)

Author: Disse 3 er vel enten implisitt eller ikke strengt nødvendige?

```suggestion
```

> - [Docker](https://www.docker.com/get-started)

Author: ```suggestion
```

> - [Docker Compose](https://docs.docker.com/compose/install/)

Author: ```suggestion
```
